packages/block-library/src/social-link/index.php

```diff
@@ -16,11 +16,10 @@
  */
 function render_block_core_social_link( $attributes, $content, $block ) {
 	$open_in_new_tab = $block->context['openInNewTab'];
-
-	$service    = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
-	$url        = ( isset( $attributes['url'] ) ) ? $attributes['url'] : false;
-	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service );
-	$class_name = isset( $attributes['className'] ) ? ' ' . $attributes['className'] : false;
+	$service         = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
```

Comment:
I think this and the others could be updated to use `array_key_exists( 'service', $attributes )`. That seems to be a cleaner way that some of the other core blocks now use.